README.md

```diff
@@ -20,26 +20,24 @@ for more information.
 
 This project builds as part of the LLVM External Projects facility (see
 documentation for the `LLVM_EXTERNAL_PROJECTS` config setting). There are many
-ways to set this up but the following is recommended. It is left to the reader
-to adapt paths if deviating. We assume below that projects are checked out to
-`$HOME/src`.
+ways to set this up. We recommend using our `configure.py` script below.
+
+It is left to the reader to adapt paths if deviating. We assume below that
+projects are checked out to `$HOME/src`.
```

Comment:
Could we maybe invert the dep. chain here?

Basically ask the user to define the `LLVM_SOURCE_DIR`, `IREE_LLVM_SANDBOX_SOURCE_DIR` and `IREE_LLVM_SANDBOX_BUILD_DIR` and merely give as an example the following:

*   `LLVM_SOURCE_DIR`: $HOME/src/llvm-project
*   `IREE_LLVM_SANDBOX_SOURCE_DIR`: $HOME/src/iree-llvm-sandbox
*   `IREE_LLVM_SANDBOX_BUILD_DIR`: $HOME/src/iree-llvm-sandbox/build

Then our instructions should never mention the `$HOME` and `src magic constants anymore.